src/renderer/component/router/view.jsx

```diff
@@ -18,16 +18,21 @@ import InvitePage from 'page/invite';
 import BackupPage from 'page/backup';
 import SubscriptionsPage from 'page/subscriptions';
 
-const route = (page, routesMap) => {
+const route = (props, page, routesMap) => {
   const component = routesMap[page];
-
-  return component || DiscoverPage;
+  if (!component) {
+        props.doNotify({
+        message: __('Invalid page requested'),
+        displayType: ['snackbar'],
```

Comment:
can you try running `yarn format`? These two lines above should be indented once more.

Reply:
fixed this one, there may be some others from previous commits. If you want, run prettier on this branch please.